app/views/category/_form.html.erb

```diff
@@ -0,0 +1,9 @@
+<div class="form-group offset-md-2 col-8 offset-md-2">
+    <label for="formGroupExampleInput">Title</label>
+    <input type="text" class="form-control" id="formGroupExampleInput" placeholder="Title input">
+</div>
+<div class="form-group offset-md-2 col-8 offset-md-2">
+    <label for="exampleFormControlTextarea1">Description</label>
+    <textarea class="form-control" id="exampleFormControlTextarea1" rows="3"></textarea>
+</div>
+<%= link_to "Submit", new_path, class: "btn btn-outline-danger w-50 offset-3 col-9 offset-2 mt-4" %>
```

Comment:
please make use of the proper Rails form tags for this one,
example code:
```
<%= form_for(@category) do |f| %>
         <%= f.text_field :title, placeholder: "Lesson Title", class: "form-control" %>
         <%= f.text_field :description, placeholder: "Lesson Description", class: "form-control" %>
         <%= f.submit class:"btn btn-outline-danger w-50 offset-3 col-9 offset-2 mt-4" %>
<% end %>
```
 and add `@category = Cateogry.new` inside the categories_controller.rb